test/test_campaigns.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+
+from datetime import datetime
+from sendgrid.helpers.campaigns import *
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
+
+
+class TestCampaignObject(unittest.TestCase):
+
+    def test_set_title(self):
+        c_name = "Test Campaign"
+        camp = Campaign(title=c_name)
+        self.assertEqual(camp.title, "Test Campaign")
+
+    def test_campaign_get(self):
+        data = {
+            "categories": [
+                "summer line"
+            ],
+            "custom_unsubscribe_url": "test.url",
+            "editor": "code",
+            "html_content": "<html><head><title></title></head><body><p>Check out our summer line!</p></body></html>",
+            "ip_pool": "test_ips",
+            "list_ids": [2, 3],
+            "plain_content": "Check out our summer line!",
+            "segment_ids": [2, 3],
+            "sender_id": 4324,
+            "subject": "New Products for Summer!",
+            "title": "May Newsletter",
+            "suppression_group_id": 5123
+        }
+        camp = Campaign(**data)
+        self.assertDictEqual(camp.get(), data)
```

Comment:
`self.assertEqual` takes care of this already?

(Just below this link)
* https://docs.python.org/3/library/unittest.html?highlight=assertequal#unittest.TestCase.addTypeEqualityFunc

> The list of type-specific methods automatically used by assertEqual() are summarized in the following table. Note that it’s usually not necessary to invoke these methods directly.

Method | Used to compare | New in
-- | -- | --
assertMultiLineEqual(a, b) | strings | 3.1
assertSequenceEqual(a, b) | sequences | 3.1
assertListEqual(a, b) | lists | 3.1
assertTupleEqual(a, b) | tuples | 3.1
assertSetEqual(a, b) | sets or frozensets | 3.1
assertDictEqual(a, b) | dicts | 3.1